source/reference/projects.rst

```diff
@@ -0,0 +1,186 @@
+======================
+ Projects - Reference
+======================
+
+
+.. seealso::
+
+   :doc:`Documentation on projects </file_management/projects>`
+      Explains how to work with projects.
+
+
+Project information is saved in meta files
+that can be edited
+to allow further configuration
+than just adding or removing folders.
+You can edit the project file
+of the currently active non-anonymous project
+via the **Project → Edit Project** menu.
+
+
+File Format
+===========
+
+=============  ===========================================
+**Format**     JSON (with comments)
+
+**Extension**  ``.sublime-project``
+
+**Name**       Any
+
+**Location**   Anywhere on your filesystem
+
+**Content**    Metadata for projects
+=============  ===========================================
+
+Example
+*******
+
+Project metadata is split across three topmost sections:
+
+- ``folders``, for the included folders;
+- ``settings``, for project-specific settings; and
+- ``build_systems``, for project-specific build systems.
+
+.. code-block:: json
+   :emphasize-lines: 2,14,18
+   :linenos:
+
+   {
+       "folders":
+       [
+           {
+               "path": "src",
+               "folder_exclude_patterns": ["backup"]
```

Comment:
I thin these patterns are merged with the global ones; they don't overwrite them. I think we should indicate this.